sites/main-site/src/content/blog/2025/advisories.mdx

```diff
@@ -0,0 +1,268 @@
+embedHeaderImage: true
+authors:
+    - "MatthiasZepper"
+label:
+    - "maintainers"
+---
+
+import Profile from "@components/GitHubProfilePictureExtended.svelte";
+import { Image } from "astro:assets";
+import AdvisorySidebarHeader from "@components/advisory/AdvisorySidebarHeader.astro";
+
+<figure>
+    <figcaption>
+        With Advisories, important information about technical issues in nf-core pipelines — like regressions,
+        incompatibilities, or security alerts — now travels faster and farther than ever.
+    </figcaption>
+</figure>
+
+## Background and motivation
+
+May 2025. At least in our corner of the world, Sweden, spring had just arrived. Daylight lingered a little longer, and the first flowers carefully stretched toward the sun. But just as nature hit its stride, nf-core pipelines didn’t.
+
+As the world shook off winter’s sleep, a configuration issue in our pipeline template emerged from hibernation, too. Soon, nf-core's Slack channels and GitHub issues were buzzing like a disturbed hive — not with bees, but with confused and concerned users.
+
+With the release of Nextflow version 25.04, our software had decided to join the seasonal chaos. Fortunately, identifying the culprit didn’t take long. A subtle configuration bug — previously tolerated up through Nextflow 24.10 — now had wide-reaching effects when pipelines were executed with the newly released version 25.04.
+
+Fixing the configuration for the future was one thing; reaching everyone affected was another. Like many open-source projects, we don’t have a clear picture of who’s using our code, which meant we had no direct line to those caught off guard.
+
+To make matters trickier, resolving the issue fully required releasing new versions of the affected pipelines — a process that can easily take weeks or even months to complete across all of them. In the meantime, users needed to override the default, buggy configuration. Simple enough — if only they knew. We had the fix in hand, but no easy way to get it into all the hands that needed it. It took weeks before support requests for that issue finally started to taper off.
+
+The experience left us with a clear lesson: identifying and fixing bugs is only part of the equation — communicating effectively with a diverse, globally distributed user base is just as critical. So, we made a change.
+
+As a direct response, we introduced a new central communication channel on the nf-core website, along with an [RSS feed](https://nf-co.re/advisories/rss.xml) that users can subscribe to. This will serve as our go-to platform for broadcasting important technical updates — especially those that affect released pipelines. Hopefully, we won’t need to use it too often (because that would mean something major has gone wrong again), but when we do, [we now have a more reliable way to get the word out](https://nf-co.re/advisories/process-shell-configuration).
```

Comment:
I would move this below the 'Hello advisories' section

Reply:
Or condense this pargraph into the next sentence, they are sort of transmitting the same information